lib/events/dynamoevents/dynamoevents.go

```diff
@@ -60,6 +62,24 @@ type Config struct {
 	UIDGenerator utils.UID
 	// Endpoint is an optional non-AWS endpoint
 	Endpoint string `json:"endpoint,omitempty"`
+
+	// EnableContinuousBackups is used to enable (or disable) PITR (Point-In-Time Recovery).
+	EnableContinuousBackups bool
+
+	// EnableAutoScaling is used to enable (or disable) auto scaling policy.
```

Comment:
to enable (or disable) auto scaling.
Policy would define the type of (auto) scaling.